packages/react-ds/src/components/Alert/index.tsx

```diff
@@ -58,16 +54,21 @@ export const Alert: React.FunctionComponent<
   closable,
   onClose,
   type = '',
+  transparent = false,
```

Comment:
maybe we should use a prop called `variant` to provide more options to use together alert

````
variant: 'transparent' | 'filled'
```

Reply:
made on 660a5446171c07d6d9ca3645380979bdd937afcf